tests/curl/test_curl_encoding.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+set -e -o pipefail
+
+. $builddir/tests/test_common.sh
```

Comment:
Try to use relative path instead of `$builddir/tests`

```suggestion
. ../test_common.sh
```